.ci/env/tbb.sh

```diff
@@ -0,0 +1,60 @@
+# Function to display help
```

Comment:
Currently location and naming assumes that it's widely applicable for all platforms, while it's hardcoded for ARM. + there is following download script that takes prebuild tbb versions- https://github.com/oneapi-src/oneDAL/blob/main/dev/download_tbb.sh

Might eb it worth aligning or might be merging them?